File: src/classes/api/req_res/add-nlog-response.ts
'use strict'

import { Nlog } from '@/classes/datas/nlog'
import { GkillAPIResponse } from '../gkill-api-response'
import { Kyou } from '@/classes/datas/kyou'

export class AddNlogResponse extends GkillAPIResponse {

    added_nlog: Nlog

    added_kyou: Kyou | null

    constructor() {
        super()
        this.added_nlog = new Nlog()
        this.added_kyou = null
    }

}

File: src/app/gkill/api/req_res/add_nlog_response.go
// ˅
package req_res

import (
	"github.com/mt3hr/gkill/src/app/gkill/api/message"
	"github.com/mt3hr/gkill/src/app/gkill/dao/reps"
)

// ˄

type AddNlogResponse struct {
	// ˅

	// ˄

	Messages []*message.GkillMessage `json:"messages"`

	Errors []*message.GkillError `json:"errors"`

	AddedNlog *reps.Nlog `json:"added_nlog"`

	AddedNlogKyou *reps.Kyou `json:"added_nlog_kyou"`

	// ˅

	// ˄
}

// ˅

// ˄
